src/conversion/model-to-view-converters.js

```diff
@@ -105,110 +133,98 @@ export function insertText() {
  * @returns {Function} Insert element event converter.
  */
 export function insertUIElement( elementCreator ) {
-	return ( evt, data, consumable, conversionApi ) => {
+	return ( evt, data, conversionApi ) => {
```

Comment:
Why there are no consumables here? That's strange that if you wants to prevent child consuming here you need to do something different that in other converteres.

Reply:
Consumables are pretty awkward here after refactoring. Earlier, every item in the marker range was put in consumables (so it was multiple values unless the range was collapsed, then it was just the range).

It made sense earlier to have consumables. Right now, because the model at the moment of conversion is already changed, we cannot look at it and generate consumables as earlier. I could pass just the range there but then... what's really the difference compared to stopping the event? It would be there just to be there.

Of course, the other way would be to make it possible to consume only a part of a range and keep this information in `consumable`. But this would make the default converter more complicated and it would make writing new converters much more complicated.

If you have any good example where consumables would actually be useful, please comment. If not, I propose keeping it the way it is.